packages/react-scripts/config/modules.js

```diff
@@ -130,7 +134,39 @@ function getModules() {
     // Otherwise we'll check if there is jsconfig.json
     // for non TS projects.
   } else if (hasJsConfig) {
-    config = require(paths.appJsConfig);
+    if (hasTsSetup) {
+      const ts = require(resolve.sync('typescript', {
```

Comment:
Rather than using `hasTsSetup`, I think you could just do a try/catch here?

Reply:
Done :)